### DISCUSSION TITLE
[Feature]: Scroll to Zoom on Web

### DISCUSSION THREAD

**Discussion by jammin1911** [immich-app/immich#1690] [Feature Request]
### Feature detail

Like with Google Photos and most other photo viewers online - Mouse scroll up to zoom in on images, mouse scroll down to zoom out.

### Platform

Web

**Comment by athornfam2** [comment]
Found this issue myself and adding that I would like this feature add too.

**Comment by Iceman248** [comment]
One of the first things I noticed when evaluating it as a Google Photos replacement. Please add.

**Comment by uhthomas** [comment]
Fixed by #2577.
